src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--31">
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the modifier name should be separated from the block or element name using a single underscore `_`. You should use `calendar_start-day_sun` and `calendar_month-length_31` instead of `calendar--start-day--sun` and `calendar--month-length--31`.